src/applications/claims-status/components/ClaimsAppealsUnavailable.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import isBrandConsolidationEnabled from '../../../platform/brand-consolidation/feature-flag';
 
-const propertyName = isBrandConsolidationEnabled() ? 'VA.gov' : 'Vets.gov';
+const siteName = isBrandConsolidationEnabled() ? 'VA.gov' : 'Vets.gov';
```

Comment:
I wonder if we should make this common code? `import siteName from '../../../platform/brand-consolidation/site-name'`. It's probably useful in other apps, too.

Reply:
Ah, yeah, that makes sense. Good call.